src/pages/Reviews.jsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+import axios from 'axios'
+import ReviewItem from '../components/ReviewItem'
+
+export default function Reviews() {
+    
+    const [allReviews, setAllReviews] = useState([])
+
+    useEffect(() => {
+        axios.get('https://shadia-nc-games.herokuapp.com/api/reviews')
```

Comment:
Could you refactor this so that all your api request live in an api.js file. 
it will make it a lot neater and easier to maintain if all the requests live somewhere else, like the lecture on monday